package.json

```diff
@@ -2,21 +2,22 @@
   "name": "patternfly-seed",
   "version": "0.0.1",
   "description": "An open source build scaffolding utility for web apps.",
-  "main": "index.js",
+  "main": "server.js",
   "repository": "https://github.com/patternfly/patternfly-react-seed.git",
   "license": "MIT",
   "private": true,
   "scripts": {
-    "prebuild": "yarn clean",
+    "prebuild": "npm run clean",
     "build": "webpack --config webpack.prod.js",
-    "start": "webpack-dev-server --hot --color --progress --info=true --config webpack.dev.js",
+    "start": "node server.js",
+    "start:dev": "webpack-dev-server --hot --color --progress --info=true --config webpack.dev.js",
```

Comment:
Could we consider a different name than `start` for starting the express server? Maybe `start:prod`?

Reply:
Start is the default name that the node image uses when running a package.json based project - it's effectively a reserved keyword in terms of running these so we need to stick with that unfortunately.

Reply:
Got ya, ok no problem. Could we instead update the readme to mention `start:dev` instead of `start` for the basic startup?

Reply:
yep - let me take care of that

Reply:
Actually, we can just merge this and I'll include the readme updates in #78 since I'm already updating that anyway.